index.d.ts

```diff
@@ -0,0 +1 @@
+declare module 'snappyjs';
```

Comment:
```suggestion
declare module 'snappyjs' {
  export function compress(buffer: Buffer): Buffer;
  export function uncompress(buffer: Buffer, maxLength?: number): Buffer;
}
```

Reply:
Removed this completely after switching to `snappy`